cmd/cerbosctl/list/list.go

```diff
@@ -35,15 +37,21 @@ func NewListCmd(fn internal.WithClient) *cobra.Command {
 }
 
 func runListCmdF(c client.AdminClient, cmd *cobra.Command, _ []string) error {
-	opts, err := internal.GenListPoliciesFilterOptions(listPoliciesFlags)
-	if err != nil {
-		return fmt.Errorf("error generating list options: %w", err)
-	}
-	policies, err := c.ListPolicies(context.Background(), opts...)
+	policyIds, err := c.ListPolicies(context.Background())
 	if err != nil {
 		return fmt.Errorf("error while requesting policy list: %w", err)
 	}
 
+	policies := make([]*policy.Policy, 0, len(policyIds))
+	for _, chunk := range chunks(policyIds, maxPolicyPerReq) {
+		var p []*policy.Policy
+		p, err = c.GetPolicy(context.Background(), chunk...)
+		if err != nil {
+			return fmt.Errorf("error while requesting policy: %w", err)
+		}
+		policies = append(policies, p...)
+	}
+
```

Comment:
How about just iterating through the ID list and whenever you hit an index divisible by 25, pass the slice range `policyIDs[idx-25:idx]` to  `GetPolicyList` and then immediately printing the batch you  get from the server? That way you don't need to store all policies in memory (which could be quite a lot of them).

Reply:
I think this wouldn't cover the scenario where there are less than 25 policy ids in the slice if I am not misunderstanding.
I came up with the following instead;
```
for idx := range policyIds {
    if idx%maxPolicyPerReq == 0 {
        var p []*policy.Policy
	p, err = c.GetPolicy(context.Background(), policyIds[idx:minInt(idx+maxPolicyPerReq, len(policyIds)-idx)]...)
	if err != nil {
	    return fmt.Errorf("error while requesting policy: %w", err)
	}
	if err = printPolicies(cmd.OutOrStdout(), p, listPoliciesFlags.OutputFormat()); err != nil {
	    return fmt.Errorf("could not print policies: %w", err)
        }
    }
}
```

Reply:
Good point.